tests/linalg_test.py

```diff
@@ -848,9 +848,10 @@ def testTensordot(self, lhs_shape, rhs_shape, axes, dtype):
       b=[(), (3,), (2, 3)],
       dtype=float_types + complex_types,
       compute_uv=[False, True],
+      lapack_driver=["gesdd", "gesvd"],
```

Comment:
I think this version of the test should only be executed on CPU, so perhaps add `None` as an option here, and then skip this test when `lapack_driver is not None and not jtu.test_device_matches(["cpu"])`?

Reply:
I removed the test since I removed the scipy interface. The `lax` part is not tested yet as far as I see, should this be addressed in this PR or is this a general issue which has to be addressed?